src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -256,6 +276,11 @@ export const QueryResultPane = () => {
                                             var columnLength =
                                                 metadata?.resultSetSummary
                                                     ?.columnInfo?.length;
+                                            if (metadata?.isExecutionPlan) {
+                                                getExecutionPlanGraphs(
+                                                    r.rows[0][0].displayValue,
```

Comment:
can you add comments explaining what the zeros achieve?  something like `// first column's name`